morpheushelper/cogs/color_picker.py

```diff
@@ -0,0 +1,63 @@
+from PyDrocsid.translations import translations
+from discord import Embed, File, Colour
+from discord.ext import commands
+from discord.ext.commands import Cog, Bot
+import re
+from PIL import ImageColor, Image
+import io
+import colorsys
+
+
+class ColorPickerCog(Cog):
+    RE_HEX = re.compile(r'^#([a-fA-F0-9]{6}|[a-fA-F0-9]{3})$')
+    RE_RGB = re.compile(r'^rgb\(([0-9]{1,3})\, ?([0-9]{1,3})\, ?([0-9]{1,3})\)$')
+    RE_HSV = re.compile(r'^hsv\(([0-9]{1,3})\, ?([0-9]{1,3})\, ?([0-9]{1,3})\)$')
+    RE_HSL = re.compile(r'^hsl\(([0-9]{1,3})\, ?([0-9]{1,3})\, ?([0-9]{1,3})\)$')
+
+    def __init__(self, bot: Bot):
+        self.bot = bot
+
+    @commands.command(name="colorpicker", aliases=["cp", "color"])
+    async def colorpicker(self, ctx, *, color: str):
+        if color_re := self.RE_HEX.match(color):
+            color_hex = color_re.group(1)
+            rgb: tuple[int] = ImageColor.getcolor(color, "RGB")
+            hsv: tuple[int] = ImageColor.getcolor(color, "HSV")
+            # skipcq: PYL-E1120
+            hsl = tuple(map(int, colorsys.rgb_to_hls(*rgb)))
+        elif color_re := self.RE_RGB.match(color):
+            rgb = (int(color_re.group(1)), int(color_re.group(2)), int(color_re.group(3)))
+            color_hex = '%02x%02x%02x' % rgb
+            hsv: tuple[int] = ImageColor.getcolor(f'#{color_hex}', "HSV")
+            # skipcq: PYL-E1120
+            hsl = tuple(map(int, colorsys.rgb_to_hls(*rgb)))
+        elif color_re := self.RE_HSV.match(color):
+            hsv: tuple[int] = (int(color_re.group(1)), int(color_re.group(2)), int(color_re.group(3)))
+            # skipcq: PYL-E1120
+            rgb = tuple(map(int,  colorsys.hsv_to_rgb(*hsv)))
+            color_hex = '%02x%02x%02x' % rgb
+            # skipcq: PYL-E1120
+            hsl = tuple(map(int, colorsys.rgb_to_hls(*rgb)))
+        elif color_re := self.RE_HSL.match(color):
+            hsl: tuple[int] = (int(color_re.group(1)), int(color_re.group(2)), int(color_re.group(3)))
+            # skipcq: PYL-E1120
+            rgb = tuple(map(int,  colorsys.hls_to_rgb(*hsl)))
+            # skipcq: PYL-E1120
+            hsv = tuple(map(int, colorsys.rgb_to_hsv(*rgb)))
```

Comment:
Please consider writing the comment for the linter behind the actual line, like so:
```python
hsv = tuple(map(int, colorsys.rgb_to_hsv(*rgb)))  # skipcq: PYL-E1120
```